elfi/methods/parameter_inference.py

```diff
@@ -775,12 +775,12 @@ def __init__(self, model, target_name=None, batch_size=1, initial_evidence=None,
             initial_evidence = len(params)
             self._n_precomputed = initial_evidence
 
-        if initial_evidence < n_initial_required:
-            raise ValueError('Need at least {} initialization points'.format(n_initial_required))
-
         if initial_evidence % self.batch_size != 0:
             raise ValueError('Number of initial evidence must be divisible by the batch size')
 
+        if initial_evidence < n_initial_required:
+            logger.warning('For best performance please use at least {} initialization points'.format(n_initial_required))
```

Comment:
Does it run now with 0 initial points, that would be good to check. Also perhaps just say something like we recommend some initialization points, since there is no guarantee for the best performance :)